components/usage/pkg/db/workspace_instance.go

```diff
@@ -70,13 +70,17 @@ func (i *WorkspaceInstance) TableName() string {
 // - instances which only just started in the period specified
 func ListWorkspaceInstancesInRange(ctx context.Context, conn *gorm.DB, from, to time.Time) ([]WorkspaceInstance, error) {
 	var instances []WorkspaceInstance
+	var instancesInBatch []WorkspaceInstance
 	tx := conn.WithContext(ctx).
 		Where(
 			conn.Where("stoppedTime >= ?", TimeToISO8601(from)).Or("stoppedTime = ?", ""),
 		).
 		Where("creationTime < ?", TimeToISO8601(to)).
 		Where("startedTime != ?", "").
-		Find(&instances)
+		FindInBatches(&instancesInBatch, 1000, func(tx *gorm.DB, batch int) error {
```

Comment:
What's the `batch` parameter intended to be used for? If it's not used can we replace it with `_` ?

Reply:
https://gorm.io/docs/advanced_query.html#FindInBatches

Will make it unused.